<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelSimuMain</class>
 <widget class="QWidget" name="ChannelSimuMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
            background-color: #336666;
        }</string>
  </property>
  <widget class="QLabel" name="welcomeBG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
            color: #2196F3;          /* 文本颜色 */
            font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
            font-size: 25px;         /* 字号 */
            font-weight: bold;       /* 加粗 */
        }</string>
   </property>
   <property name="text">
    <string>信道模拟场景选择</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="modelAddBT">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(240, 255, 255);
border:2px rgb(243, 243, 243);
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/image/circle_plus.png</normaloff>:/res/image/circle_plus.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="modelDelBT">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(240, 255, 255);
border:2px rgb(243, 243, 243);
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/image/circle_remove.png</normaloff>:/res/image/circle_remove.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="model1BT">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>短波地波山地</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="model2BT">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>短波地波平原</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="model3BT">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>短波地波海上</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="model4BT">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>短波天波单跳</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="model5BT">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>短波天波多跳</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="model6BT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>短波天波单音</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="model7BT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>短波天波仰角</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="diy1BT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>自定义1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="diy2BT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>自定义2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="diy3BT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 基础RadioButton样式 */
QRadioButton {
    spacing: 6px;  /* 文本与按钮间距 */
    color: #4CAF50;          /* 文本颜色 */
    font-family: &quot;Microsoft YaHei&quot;;  /* 字体 */
    font-size: 16px;         /* 字号 */
    font-weight: bold;       /* 加粗 */
}</string>
   </property>
   <property name="text">
    <string>自定义3</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
